### PR TITLE
Implemented OpenCv stages

### PR DESCRIPTION
**Pull request by Casper-NS**
Created opencv stages that can be used for comparison/debugging.
Also fixed time logging

### CODE REVIEW

File: app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/openCVPlaceholders/OpenCVCannyStage.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    override fun update() {
+        Utils.bitmapToMat(bitmap, img)
+
+        var edges = Mat(img.size(), CvType.CV_8UC1)
+
+        Imgproc.Canny(edges, edges, 80.0, 130.0)
+
+        Utils.matToBitmap(edges, outputBitmap)
+    }
```
Comment (Mati-AAU) [inline app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/openCVPlaceholders/OpenCVCannyStage.kt:26]: what is img used for? seems unnecessary/unefficient

File: app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/openCVPlaceholders/OpenCVGaussianBlurStage.kt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    override fun update() {
+        Utils.bitmapToMat(bitmap, img)
+
+        var blurredImg = Mat(img.size(), CvType.CV_8UC1)
+
+        Imgproc.GaussianBlur(img, blurredImg, org.opencv.core.Size(5.0,5.0), 1.0, 1.0)
+
+        Utils.matToBitmap(blurredImg, outputBitmap)
+    }
```
Comment (Mati-AAU) [inline app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/openCVPlaceholders/OpenCVGaussianBlurStage.kt:35]: can probably be simpler like canny

**Review by Mati-AAU — APPROVED** [review]
lgtm

File: app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/openCVPlaceholders/OpenCVCannyStage.kt (hunk 1 above)
Comment (Casper-NS) [reply app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/openCVPlaceholders/OpenCVCannyStage.kt]: fixed

**Review by Mati-AAU — APPROVED** [review]
lgtm

**Review by Mati-AAU — APPROVED** [review]
lgtm still

**Pull request merged**